posix/clone

```diff
@@ -20,12 +24,12 @@ fi
 # known hosts file.
 
 if [[ ! -z "${SSH_KEY}" ]]; then
-	mkdir /root/.ssh
-	echo -n "$SSH_KEY" > /root/.ssh/id_rsa
-	chmod 600 /root/.ssh/id_rsa
+	mkdir /${HOME}/.ssh
```

Comment:
since the HOME path will be absolute I recommend removing the path prefix which will avoid a `//` at the start of the path

```diff
-/${HOME}/.ssh
+${HOME}/.ssh
```